app/src/main/java/com/dimagi/biometric/fragments/FaceMatchFragment.java

```diff
@@ -76,4 +95,21 @@ public void onCancelled() {
     public void onTimedout(byte[] faceImage) {
         handleErrorMessage("Failed to capture face image: Timed out");
     }
+
+    @Override
+    protected ParamManager getParams() {
+        Intent intent = requireActivity().getIntent();
+        ParamManager params = new ParamManager();
+        params.setPitch(intent.getIntExtra(ParamConstants.PITCH_NAME, ParamConstants.PITCH_DEFAULT));
```

Comment:
The defaults aren't necessarily the minimum values, but moreso the recommended default values from Tech5's documentation. You are correct in that these values can be set, see my [previous comment](https://github.com/dimagi/biometric-android/pull/4#discussion_r1312673536) for more context.

I think users more often than not will probably opt to simply use the defaults on most of these parameters, although we could still give a warning if values are given that have gone out of the allowed range. I think it is important that this not be a blocking error however, as this would first require a CommCare app update which will completely stop users from completing their workfows.

Reply:
sounds good